doc/lsst.pipe.tasks/tasks/lsst.pipe.tasks.makeWarp.MakeWarpTask.rst

```diff
@@ -0,0 +1,123 @@
+.. lsst-task-topic:: lsst.pipe.tasks.makeWarp.MakeWarpTask
+
+############
+MakeWarpTask
+############
+
```

Comment:
You also want the summary sentence and intro paragraph from `MakeWarpTask` here, and all the stuff below in the [In Depth](https://developer.lsst.io/stack/task-topic-type.html#in-depth-section) section (using subsections instead of sections).

We probably do want a [Processing Summary](https://developer.lsst.io/stack/task-topic-type.html#processing-summary-section) selction, too: why don't you make a placeholder for it (file a ticket and link it in the new empty section), so you don't have to figure it out for an unfamiliar task.